monkey/monkey_island/cc/ui/src/components/ui-components/IslandResetModal.tsx

```diff
@@ -96,27 +96,27 @@ const IslandResetModal = (props: Props) => {
   function clearSimulationData(callback: () => void) {
     auth.authFetch('/api/clear-simulation-data', {method: 'POST'})
       .then(res => {
-        if (res.status === 200) {
+        if (res.ok) {
           callback();
         }
       })
   }
   function resetAll() {
     auth.authFetch('/api/reset-agent-configuration', {method: 'POST'})
       .then(res => {
-        if (res.status === 200) {
+        if (res.ok) {
             return auth.authFetch('/api/clear-simulation-data', {method: 'POST'})
         }})
         .then(res => {
-          if (res.status === 200) {
+          if (res.ok) {
               return auth.authFetch('/api/propagation-credentials/configured-credentials', {method: 'PUT', body:'[]'})
           }})
       .then(res => {
-        if (res.status === 200) {
-            return auth.authFetch('/api/island-mode', {method: 'POST', body: '{"mode": "unset"}'})
+        if (res.ok) {
+            return auth.authFetch('/api/island/mode', {method: 'PUT', body: '{"mode": "unset"}'})
         }})
       .then(res => {
-        if (res.status !== 200) {
+        if (!res.ok) {
```

Comment:
`PUT /api/island/mode` returns 204, not 200. Does `res.ok` capture this?

Reply:
`res.ok` is if the response was successful, I am not sure if we should rely on the status code as previously was comparing `200` but we return `204` and while the response was `ok` the reseting of the island mode was not working.

Reply:
We're good: 
![image](https://user-images.githubusercontent.com/19957806/182180242-823fe36f-e74c-47a3-9ed1-c5c54f67f8ed.png)